docs/installation/self-hosting/self-hosted-installation.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Self hosting
+description: 'Installing Agenta on your cloud provider'
+---
+
+## AWS
+We provide a Terraform module to deploy Agenta on AWS. You can find it [here](https://github.com/Agenta-AI/agenta/tree/main/self-host/aws).
+The terraform module will simply create an instance and the resources needed to run Agenta on it.
+You can customize the module according to your needs.
+
```

Comment:
you did not mention the step of creating a key. 
the documentation should be self explanatory and reading the code should be needed